src/service/api.tsx

```diff
@@ -1,16 +1,18 @@
 import { API_KEY, API_IMG_URL } from '../constants/api';
 import axios from '../axios/url';
 
-import { Theme, MovieCard } from '../models/index';
+import { Theme, MovieCard, HistoryBar } from '../models/index';
+
 const apiService: {
   storeKey: string;
-  store: { history: number[]; favorites: number[]; theme: any };
+  store: { history: number[]; favorites: number[]; theme: number; historyBar: number };
```

Comment:
Wonder why not Theme and HistoryBar?

Reply:
are you mean to make these name from capital letter?

Reply:
I meant types for the values